Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/Create/CreateStreetcodeHandler.cs

```diff
@@ -151,9 +157,22 @@ await _repositoryWrapper.StreetcodeImageRepository.CreateRangeAsync(imagesIds.Se
 
     private async Task AddImagesDetails(IEnumerable<ImageDetailsDto>? imageDetails)
     {
+        if(imageDetails.IsNullOrEmpty())
+        {
+            throw new HttpRequestException("There is no valid imagesDetails value", null, System.Net.HttpStatusCode.BadRequest);
+        }
+
         if(imageDetails == null)
         {
-            return;
+            throw new HttpRequestException("There is no valid imagesDetails value", null, System.Net.HttpStatusCode.BadRequest);
+        }
```

Comment:
variable.IsNullOrEmpty() is more versatile and commonly used when you want to check if the variable is null or empty, regardless of its type, whereas var == null is specifically for checking if a variable is null. You basically making additional check that never be neither completed or true. Delete second statement

Reply:
I have already removed second statement